config/development.yml

```diff
@@ -1,4 +1,22 @@
-webAppHost: "http://localhost:8080"
+webAppHost: 'http://localhost:8080'
+
+jwt:
+  token: 'JWT_TOKEN'
+
+communication:
+  services: ['twilio', 'sendgrid']
+
+twilio:
+  verify:
+    accountSid: 'AC.TWILIO_ACCOUNT_SID'
+    authToken: 'TWILIO_AUTH_TOKEN'
+  messaging:
+    messagingServiceSid: 'TWILIO_MESSAGING_SERVICE_SID'
+
+sendgridApiKey: 'SG.SENDGRID_API_KEY'
 
 mongoDb:
-  uri: "mongodb://localhost:27017//boilerplate-mern-dev"
+  uri: 'mongodb://localhost:27017/boilerplate-mern-dev'
+
+server:
+  port: '8080'
```

Comment:
This is being picked up from `default` already. Adding it here will have no effect. Advise to remove this.

Reply:
Removed this already